src/components/NewCommentForm.tsx

```diff
@@ -21,6 +18,10 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
     body: '',
   });
 
+  const post = useAppSelector(state => state.posts?.selectedPost);
```

Comment:
```suggestion
  const post = useAppSelector(state => state.posts.selectedPost);
```

looks like redundant